test/reservation_test.rb

```diff
@@ -0,0 +1,28 @@
+require_relative "test_helper"
+
+describe HotelBooking::Reservation do
+  before do
+    @duration = HotelBooking::Duration.new(
+      start_date: "2019-09-01",
+      end_date: "2019-09-04",
+    )
+    @cost = HotelBooking::Reservation.new(id: 1,
+                                          customer_id: 2,
+                                          room: 34,
+                                          duration: @duration)
+  end
+
+  describe "cost" do
```

Comment:
You can put multiple `it` blocks in one `describe` block. Right now, you have two `describe` blocks that are both named `"cost"`... instead, consider moving the `it` blocks under the same `describe` block.